### DISCUSSION TITLE
GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE_V2 vs Finances API

### DISCUSSION THREAD

**Discussion by kmizzi** [amzn/selling-partner-api-models#3474] [General]
Aside from have data more quick to access with Finances API, is there any other advantage to using Finances API over GET_V2_SETTLEMENT_REPORT_DATA_FLAT_FILE_V2 report?

Even with getting information quicker with Finances API, it becomes difficult to manage when there is no unique id for each event.  As a result, it is better to wait until the Financial Event Group becomes closed and then this takes away the advantage.  You also have to manage the mappings of the 3 common elements of date, description, and amount and a few of the event types don't have a date.

**Comment by ghost** [comment]
Hey hi, I am looking for v2 report for my electronic payments any idea on how can i get that data?
